admin/src/js/controllers/main.js

```diff
@@ -9,9 +9,14 @@ angular.module('controllers').controller('MainCtrl',
   ) {
     'ngInject';
     $translate.use('en');
-    Auth('can_configure').catch(function() {
+    $scope.authorized = false;
+    Auth('can_configure')
+    .then(function() {
+      $scope.authorized = true;
+    })
+    .catch(function() {
       $log.error('Insufficient permissions. Must be either "admin" or "nationalAdmin".');
-      $window.location.href = '../../../login';
+      $window.location.href = '../../login';
```

Comment:
Use `Location.path` from the Location service. It gets an absolute path instead of a relative one so you won't get any off by one errors. Also it redirects to the webapp which is probably better than `/login`.